lib/rspec/rails/fixture_support.rb

```diff
@@ -50,6 +50,10 @@ def self.proxy_method_warning_if_called_in_before_context_scope(method_name)
             end
           end
 
+          def name
```

Comment:
If this is only needed on master, we should surround it with a version check, e.g:
```suggestion
          if Rails.version.to_f > 6
            def name
              @example
            end
          end
```